packages/react-core/src/demos/Wizard/WizardDemo.md

```diff
@@ -91,16 +71,40 @@ class BasicWizardDemo extends React.Component {
           aria-describedby="wiz-modal-demo-description"
           aria-labelledby="wiz-modal-demo-title"
         >
-          <WizardDeprecated
-            navAriaLabel={`${title} steps`}
-            mainAriaLabel={`${title} content`}
-            titleId="wiz-modal-demo-title"
-            descriptionId="wiz-modal-demo-description"
-            title="Simple wizard in modal"
-            description="Simple wizard description"
-            steps={steps}
+          <Wizard
```

Comment:
Codesandbox is warning me that there is a `key` problem in this Wizard: 
```
Check the render method of `WizardToggle`. See https://fb.me/react-warning-keys for more information.
    in Fragment (created by WizardToggle)
    in WizardToggle (created by WizardInternal)
    in WizardInternal (created by Wizard)
    in div (created by Wizard)
    in WizardContextProvider (created by Wizard)
    in Wizard (at index.js:62)
    in div (created by ModalBox)
  ```

Reply:
and a second `key` problem: 
```
Check the render method of `WizardNavInternal`. See https://fb.me/react-warning-keys for more information.
    in WizardNavItem (created by WizardNavInternal)
    in WizardNavInternal (created by WizardInternal)
    in div (created by WizardToggle)
    in div (created by WizardToggle)
    in WizardToggle (created by WizardInternal)
    in WizardInternal (created by Wizard)
    in div (created by Wizard)
    in WizardContextProvider (created by Wizard)
    in Wizard (at index.js:62)
 ```

Reply:
These warnings are appearing in all these demos - and are manifesting as the following error in the workspace:
```
Uncaught Error: Minified React error #423; visit https://reactjs.org/docs/error-decoder.html?invariant=423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Sl (5472.596e471a.bundle.js:2:3393845)
    at bc (5472.596e471a.bundle.js:2:3381463)
    at _c (5472.596e471a.bundle.js:2:3381391)
    at mc (5472.596e471a.bundle.js:2:3381254)
    at rc (5472.596e471a.bundle.js:2:3378061)
    at oc (5472.596e471a.bundle.js:2:3376613)
    at w (5472.596e471a.bundle.js:2:3429769)
    at MessagePort.I (5472.596e471a.bundle.js:2:3430301)
 ```

Reply:
Looks like there was one WizardStep that I mistyped the `id` prop as just `d`, should be fixed in the latest push.